### ISSUE TITLE
more a question

### ISSUE THREAD

**Issue by NikolaasBender** [mosheliv/tensorflow-1.8.0-without-avx#2]
How did you compile this wheel? I've been trying to get this right and every time I run into issues with bazel, or ffmpeg, or any number of other things. how did you do it?

**Comment by mosheliv** [comment]
It was a real pain, that is why I posted it. Bazel is problematic with
saving its cache files. If you have a smallish SSD, you need to instruct
bazel to use a different directory for its products/cache (I think this was
my biggest problem). I remember having some trouble with ffmpeg but it was
not major. Sorry can't remember the details. Let me know what errors do you
get and I will try to help. I am currently using 1.10 though on a machine
with avx

**Comment by NikolaasBender** [comment]
So I'm trying to compile tf 1.9 without avx (old silicon reppin!). The issue comes with a reference to 'clock_gettime'.

**Comment by NikolaasBender** [comment]
How about jemalloc, have you ever seen issues with that?

**Comment by mosheliv** [comment]
nope, what linux are you compiling it on? the jemalloc rings a bell, might
be just something you need to install.

**Comment by NikolaasBender** [comment]
Using 18.04 I'm trying to just not use jemalloc. I'm using a docker build script with a config file to help with this. But I just keep trying different things, hopefully some combination will get it to work

**Comment by mosheliv** [comment]
I would build it on something that actually existed when 1.9 was around,
like 16.04. It usually works better as it is the de-facto standard., so the
build has been tested on it
